odoo/release.py

```diff
@@ -12,7 +12,7 @@
 # properly comparable using normal operarors, for example:
 #  (6,1,0,'beta',0) < (6,1,0,'candidate',1) < (6,1,0,'candidate',2)
 #  (6,1,0,'candidate',2) < (6,1,0,'final',0) < (6,1,2,'final',0)
-version_info = (13, 5, 0, ALPHA, 1, '')
+version_info = ('saas~13', 5, 0, FINAL, 0, '')
```

Comment:
@kebeclibre Is this how we do it ?

Reply:
@SimonGenin 
Yes, this is how a relase is made, (that and the fork to its own branch)

You'll need to cherry-pick your commit on top of a fresh saas-13.5 (and redo a PR for that release)